backend/api/invoices/fetch.py

```diff
@@ -18,6 +18,9 @@
 from backend.models import Invoice, InvoiceItem
 
 
+default_sort_by = {"date_due": True, "id": True, "payment_status": True}
```

Comment:
"default_" is misleading since you change it below but you may think that it's a constant since it's "default"